driver/compmisc.ml

```diff
@@ -45,9 +45,10 @@ let init_path ?(dir="") native =
 
 let open_implicit_module m env =
   let open Asttypes in
-  let lid = {loc = Location.in_file "command line";
-             txt = Longident.parse m } in
-  snd (Typemod.type_open_ Override env lid.loc lid)
+  let loc = Location.in_file "command line" in
+  let lid = {loc; txt = Longident.parse m } in
+  let me = Parsetree.({pmod_desc=Pmod_ident lid; pmod_loc=loc; pmod_attributes=[]}) in
+  snd (Typemod.type_open_ ?toplevel:None Override env lid.loc me)
```

Comment:
This pattern of first creating a module expression from a Longident.t then applying `type_open_` appears quite frequently. Would it make sense to have a `type_open_id` function fot this common case?

Reply:
Alternatively, one could introduce a helper function such as:

````ocaml
let modexpr_of_lid lid = {Parsetree.pmod_desc=Pmod_ident lid; pmod_loc=lid.loc; pmod_attributes=[]}
````

Reply:
Or just use `Ast_helper.ident` as is, since type_open would use the location of the lid, not of the module_expr in that case anyway.